count_vowels.py

```diff
@@ -0,0 +1,58 @@
+"""
+Hometask 4 - Count vowels.
+
+# The English language has five vowels: A, E, I, O, and U
```

Comment:
Comment signs could be removed - just raw text of task description

Reply:
I commented out the table example because without it the linter gives errors